src/js/common/components/CardForListBody.jsx

```diff
@@ -17,6 +17,7 @@ import { CampaignActionButtonsWrapper, CampaignImageDesktop, CampaignImageDeskto
 import DesignTokenColors from './Style/DesignTokenColors';
 import HeartFavoriteToggleLoader from './Widgets/HeartFavoriteToggle/HeartFavoriteToggleLoader';
 import SvgImage from './Widgets/SvgImage';
+import extractPoliticianDetailsFromUrl from "../utils/extractPoliticianDetailsFromUrl";
```

Comment:
Hi John, please make sure to run `npm run lint` in your WebApp terminal window so you catch ESLint errors. For example, that will catch the use of double quotes when you should use single quotes.